internal/config/config.go

```diff
@@ -124,6 +125,7 @@ func Get() *Config {
 		options.SetDefault("db.host", cfg.Database.Hostname)
 		options.SetDefault("db.port", cfg.Database.Port)
 		options.SetDefault("rdsCa", cfg.Database.RdsCa)
+		options.SetDefault("db.sslmode", "verify-full")
```

Comment:
This _might_ cause an issue if you run changelog in bonfire/ephemeral env.  In the ephemeral envs, the db configuration is handled by clowder but the database's do not use ssl/tls.  So setting this to "verify-full" might cause an issue.

Reply:
I see! So how do I check whether it is in ephemeral?